env0/data_projects_test.go

```diff
@@ -0,0 +1,127 @@
+package env0
+
+import (
+	"errors"
+	"regexp"
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestProjectsDataSource(t *testing.T) {
+	project1 := client.Project{
+		Id:              "id0",
+		Name:            "my-project-1",
+		ParentProjectId: "p1",
+		Hierarchy:       "adsas|fdsfsd",
+	}
+
+	project2 := client.Project{
+		Id:   "id1",
+		Name: "my-project-2",
+	}
+
+	project3 := client.Project{
+		Id:         "id1",
+		Name:       "my-project-2",
+		IsArchived: true,
+	}
```

Comment:
Maybe just to make it more synced with real data

```suggestion
	project1 := client.Project{
		Id:              "id1",
		Name:            "my-project-1",
		Hierarchy:       "id1",		
		IsArchived: false,
	}

	project2 := client.Project{
		Id:   "id2",
		Name: "my-project-2",
		Hierarchy:       "id1|id2",
		IsArchived: false,
		ParentProjectId: "id1",
	}

	project3 := client.Project{
		Id:         "id3",
		Name:       "my-project-3",
		Hierarchy:       "id3",
		IsArchived: true,
	}
```